jellyfish/_webhook_notifier.py

```diff
@@ -3,16 +3,22 @@
 notification from jellyfish to notification structs.
 """
 
-from typing import Any, Dict
+import inspect
+from typing import Dict, Union
 
 import betterproto
 
-from jellyfish.events import ServerMessage
+import jellyfish
+from jellyfish.events._protos.jellyfish import ServerMessage
 
+server_messages = tuple(m[0] for m in inspect.getmembers(jellyfish.events, inspect.isclass))
```

Comment:
As this is type, shouldn't it be written in a different case e.g.: `SCREAMING_SNAKE_CASE`?

Reply:
absolutely